Подготовил ветку develop к практической части

DKay7 opened the pull request:
## Основные изменения
- Добавил файл `calculate.py` для расчета периметра и площади фигур
- Исправил формулу площади в файле `rectangle.py`
- Обновил документацию для файла `calculate.py`
- Исправил формулу площади в документации.

smartiqa commented on calculate.py:
```diff
@@ -0,0 +1,34 @@
+
+figs = ['circle', 'square', 'triangle']
+funcs = ['perimeter', 'area']
+sizes = {'area-traingle': 3, 'area-triangle': 3}
```
Area -> Perimeter
Traingle -> Triangle

DKay7 replied:
Поменял в коммите `ab8642d` `L-04: Calculate.py added`

smartiqa commented on calculate.py:
```diff
@@ -0,0 +1,34 @@
+import circle
```
Не совсем логично, что берем круг, квадрат и треугольник, а прямоугольник не берем. Либо давайте берем пару любых, либо все 4, либо пишем TODO комментарий насчет прямоугольника

DKay7 replied:
Так на ветке `develop` нет файла rectangle.py. Я включил все, какие есть.

DKay7 committed:
L-04: Tirangle.py fixed

An error was in triangle's area function. It was successfully fixed

DKay7 committed:
L-04: Rectangle.py added

Add rectangle.py which contans functions to compute area and perimeter
of given rectangle

DKay7 committed:
L-04: Calculate.py added

Calculate.py main designition is to compute area and/or perimeter
of the given figure

Now works only for rectangle. More figures will be added lately.

DKay7 committed:
L-04: Calculate.py updated

Update calculate.py: add support for more figures

DKay7 committed:
L-04: Calculate.py was fixed

Fix a little typo in calculate.py.

DKay7 committed:
L-04: Update docs. Add info about calculate.py

Add usage info about calculate.py to docs
